### PR TITLE
Cleanup mac osx package and make it more official

### PR DESCRIPTION
**Pull request by brentm5**
Cleaned up some of the build items and bundle the binaries in the right bin folder.  Also added a license and installer notes

### CODE REVIEW

File: omnibus/config/projects/workup.rb [hunk 1]
```diff
@@ -5,22 +5,20 @@
 # All Rights Reserved.
 #
 
+require_relative '../../../lib/workup/version.rb'
```
Comment (brentm5) [inline omnibus/config/projects/workup.rb]: Just pulled the version information from workup itself, it seemed easier

File: omnibus/config/projects/workup.rb [hunk 2]
```diff
@@ -37,3 +35,7 @@
   upgrade_code '769d8737-c798-49d1-bab0-0a31da3ee7df'.capitalize
   parameters ProjectLocationDir: project_location_dir
 end
+
+package :pkg do
+  identifier 'com.cvent.pkg.workup'
```
Comment (brentm5) [inline omnibus/config/projects/workup.rb]: I think this is a fair name for it?

File: omnibus/config/software/workup.rb [hunk 3]
```diff
@@ -15,50 +15,48 @@
...
 
 # Version manifest file
 dependency 'version-manifest'
 
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  gem 'build workup.gemspec', env: env
-  gem 'install workup*.gem', env: env
+  delete "#{name}-*.gem"
```
Comment (brentm5) [inline omnibus/config/software/workup.rb]: took this from inspec

File: omnibus/Gemfile [hunk 4]
```diff
@@ -2,11 +2,11 @@
 source 'http://rubygems.org'
 
 # Install omnibus
-gem 'omnibus', github: 'chef/omnibus', branch: 'ksubrama/gcc_investigate'
+gem 'omnibus', git: 'https://github.com/chef/omnibus.git', branch: 'ksubrama/gcc_investigate'
```
Comment (jonathanmorley) [inline omnibus/Gemfile]: Why git?

File: omnibus/config/projects/workup.rb (hunk 2 above)
Comment (jonathanmorley) [inline omnibus/config/projects/workup.rb]: yep

**Pull request merged**